Marlin/src/lcd/extensible_ui/ui_api.cpp

```diff
@@ -180,7 +180,9 @@ namespace ExtUI {
     return thermalManager.degTargetHotend(extruder - E0);
   }
 
-  float getFan_percent(const fan_t fan) { return ((float(fan_speed[fan - FAN0]) + 1) * 100) / 256; }
+  float getTargetFan_percent(const fan_t fan) { return ((float(fan_speed[fan - FAN0]) + 1) * 100) / 256; }
+
+  float getActualFan_percent(const fan_t fan) { return ((int(fan_speed[0] * (fan_speed_multiplier[0]/100.0f)) + 1) * 100) / 256; }
```

Comment:
This is only operating on fan zero, however.

Reply:
Fixed :)